app/src/main/java/simform/gitexcercise/android/utils/Validator.kt

```diff
@@ -0,0 +1,11 @@
+package simform.gitexcercise.android.utils
+
+object Validator {
+    fun isValidUserName(username: String): Boolean {
+        return username.isNotBlank() && username.length < 20
+    }
+
+    fun isValidPassword(password: String): Boolean {
+        return password.isNotBlank() && password.length in 8..20
```

Comment:
`password.length in 8..20` This already meets the requirement that password is not empty, or not?

Reply:
we still have to check for the blank if user add 9..19 blank spaces, since not checking for blank will generate issue

Reply:
Good catch